x-pack/plugins/serverless_search/kibana.jsonc

```diff
@@ -30,6 +30,7 @@
       "indexManagement",
       "searchConnectors",
       "searchHomepage",
+      "searchIndices",
```

Comment:
nit: since the plugin doesn't export any contract yet, I'd hold off declaring the dependency unless there's any specific reason to force a certain order in the execution of the plugins.